### PR TITLE
Added potential alternatives to apps

### PR DESCRIPTION
**Pull request by xplosionmind**

### CODE REVIEW

File: apps.toml [hunk 1]
```diff
@@ -238,7 +240,7 @@ url = "https://github.com/YunoHost-Apps/bonfire_ynh"
 [borg]
 category = "system_tools"
 level = 7
-potential_alternative_to = [ "Time Machine" ]
+potential_alternative_to = [ "Time Machine", "Veeam" ]
```
Comment (OniriCorpe) [inline apps.toml:243]: I would add Restic too

**Review by OniriCorpe — CHANGES_REQUESTED** [review]

File: apps.toml (hunk 1 above)
Comment (xplosionmind) [reply apps.toml:243]: I think that listing alternatives that *are* YunoHost apps would create a lot of confusion, also making alternatives listing a nightmare…

File: apps.toml [hunk 2]
```diff
@@ -418,7 +420,7 @@ url = "https://github.com/YunoHost-Apps/coin_ynh"
 [collabora]
 category = "office"
 level = 8
-potential_alternative_to = [ "Google Docs", "G Suite", "Microsoft Word", "Microsoft Excel", "Microsoft Office" ]
+potential_alternative_to = [ "Google Docs", "G Suite", "Microsoft Word", "Microsoft Excel", "Microsoft Office", "Pages" ]
```
[the remark this answers is not in the document]
Comment (xplosionmind) [reply apps.toml]: Resolved in [3a08abd](https://github.com/YunoHost/apps/pull/1982/commits/3a08abd17fb39a32c45d600a0ad56ec6b7262dd3)

**Review by alexAubin — APPROVED** [review]

File: apps.toml (hunk 1 above)
Comment (OniriCorpe) [reply apps.toml:243]: hmm yes, you're right

**Review by tituspijean — APPROVED** [review]

**Pull request merged**